src/Notifications/PtpJobConcluded.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Biigle\Modules\Ptp\Notifications;
+
+use Biigle\Report;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class PtpJobConcluded extends Notification
+{
+    /**
+     * The volume for which the PTP job was running.
+     *
+     * @var
+     */
+    protected $volume;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @param Report $report
+     * @return void
+     */
+    public function __construct(string $volume)
```

Comment:
This is the volume *name* only, right? If you want to keep it like that, this should be reflected in the variable name and comments.

You could also pass the entire volume, too, which would allow you to add a handy button to open the volume to the notification ([example](https://github.com/biigle/maia/blob/ab9dd4c3a61d81b734b87d42c2311b06839d33d1/src/Notifications/JobStateChanged.php#L63-L65)).